timewarp_gl/shaders/timewarp_shader.hpp

```diff
@@ -50,6 +53,7 @@ const char* const timeWarpChromaticFragmentProgramGLSL =
 	"	outColor.r = texture( Texture, vec3( fragmentUv0, ArrayLayer ) ).r;\n"
 	"	outColor.g = texture( Texture, vec3( fragmentUv1, ArrayLayer ) ).g;\n"
 	"	outColor.b = texture( Texture, vec3( fragmentUv2, ArrayLayer ) ).b;\n"
+	"	outColor.rgb = vec3(fragmentUv0,1);\n"
```

Comment:
This overwrites the actual color output. Even though this shader is not used, this bug shouldn't be here.

Reply:
Yep, this was for debugging. I may just remove this entire shader.

Reply:
Ah, that is for the alternative eye format. We'll remove this when that issue is closed (in another PR)